module3/Projects/Mod3Project.md

```diff
@@ -0,0 +1,121 @@
+
+Before writing code, there are two planning steps you need to complete and include here in the README.
+1. Draw a request/response diagram that includes all requests required to complete this iteration
+1. Draw a rough wireframe of the UI page(s) required to complete this iteration
+
+Add screenshots of your planning images in your project README, where it says:
+#### REPLACE THIS HEADER WITH YOUR ITERATION 2 REQUEST/RESPONSE IMAGE
+#### REPLACE THIS HEADER WITH YOUR ITERATION 2 WIREFRAME
+
+Once you've finished planning, check out a branch, and get started on the implementation.
+
+When you are finished with iteration 2, create a pull request and schedule time with a cohort-mate for some code review.  During the code review, make comments on the pull request indicating changes that you will make based on your review!
+
+### Iteration 3 - Displaying Statistics
+
+As in the Mod2 project, we are saving more than one session's worth of messages and can use some fun LINQ methods to analyze our data and display it to our users.
+
+This iteration includes the following features:
+* A separate page that displays statistics and some way to visit this page from any other page in your application
+* On the statistics page, display the following information (this is the same as the statistics from Mod 2)
+    * How many messages each user has written
+    * Users ordered by number of messages created (most to least)
+    * Most commonly used word for messages (by user and overall)
+    * The hour with the most messages
+    * Brainstorm your own interesting statistic(s)!
+
+For this iteration, it's up to you what type of planning you want to do before diving into code. You will want to consider the following, how do you want users to navigate to the statistics page?
+
+When you are finished with iteration 3, create a pull request and schedule time with a cohort-mate or your instructor for some code review.
+
+### 🌶️Iteration 4 - Edit and Delete
+
+If you have finished the first three iterations, meaning that all features are complete, tested, and you're happy the user experience, you can move on to this spicy stretch goal. 
+
+This iteration includes the following features:
+* The ability to Edit a user's info
+* The ability to Delete a user and their corresponding messages
+* The ability to Edit a message
+* The ability to Delete a message
+
+For this iteration, it's up to you what type of planning you want to do before diving into code.
+
+When you are finished with iteration 3, create a pull request and schedule time with a cohort-mate or your instructor for some code review.
+
+## Version Control Expectations
+In order to demonstrate your proficiency with version control, you will be using commits, branches and pull requests during this project.
+
+You must gather feedback from cohort-mates and/or instructors prior to merging any code into your `main` branch.
+
+You must provide feedback on at least one other student's pull request!
+

```

Comment:
Should we include some expectations around usability?  Like, we should never have to type in an address, or use the back button?

Reply:
I like this idea